src/lib/Scenes/Fair/Components/FairHeader/index.tsx

```diff
@@ -49,10 +49,61 @@ const CountdownContainer = styled.View`
   width: 100%;
 `
 
+const Bold = styled(Sans)`
+  font-weight: bold;
+`
 export class FairHeader extends React.Component<Props> {
+  getArtists(artistCounts, artistsNames) {
+    const { artists } = artistCounts
+
+    return (
+      <Flex flexDirection="row" flexWrap="wrap">
+        <Sans size="3">Works by </Sans>
+        <Bold size="3">{artistsNames.edges[0].node.name + ", "}</Bold>
```

Comment:
this would fail if `edges.length == 0` for some reason. I would store `edges` in a variable and check before getting data